EarthValley80/EarthValley80/Screen/YomojomoNews/Cells/EmptySpaceCollectionViewCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  EmptySpaceCollectionViewCell.swift
+//  EarthValley80
+//
+//  Created by LeeJiSoo on 2022/10/28.
+//
+
+import UIKit
+
+class EmptySpaceCollectionViewCell: UICollectionViewCell {
+
+    // MARK: view
+
+    private let addBackgroundView: UIImageView = {
+        let backgroundView = UIImageView()
+        backgroundView.contentMode = .scaleToFill
+        backgroundView.clipsToBounds = true
+        backgroundView.layer.cornerRadius = 30
+        // TODO: - 카테고리에 적절한 색상을 넣어야합니다.
+        backgroundView.backgroundColor = .red
+        return backgroundView
+    }()
+
+    // MARK: method
+
+    func setupLayout() {
+        self.addSubview(addBackgroundView)
+        self.addBackgroundView.constraint(to: self,
+                                          insets: UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0))
```

Comment:
p2;

.zero로 변경해서 쓸 수 있을 거 같네요.🙌

```suggestion
                                          insets: UIEdgeInsets.zero
```

Reply:
이부분도 알려주신 부분인데, 리뷰 순서가 엉켰네요!! 이제부터는 꼭 적용시킬게요 🫰
[fac281f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-EarthValley80/pull/41/commits/fac281f8df70565ac1be76290afc4b13a7349a30)